choco-remixer/public/Invoke-RepoCheck.ps1

```diff
@@ -82,13 +82,20 @@
             #pwsh considers 3xx response codes as an error if redirection is disallowed
             if ($PSVersionTable.PSVersion.major -ge 6) {
                 try {
-                    Invoke-WebRequest -UseBasicParsing -Uri $publicPage.feed.entry.content.src -MaximumRedirection 0 -ea Stop
+                    $redirectpage = Invoke-WebRequest -UseBasicParsing -Uri $publicPage.feed.entry.content.src -MaximumRedirection 0 -ea Stop
```

Comment:
Why are these extra lines being added? From what I understand, the Invoke-WebRequest call should always error in powershell core.

Reply:
It does Error when the Webserver does a redirect like the Powershell Galery does.
But Nexus does not error as it provides the zip File at this stage without a redirect